docs/src/components/banner/AppHeaderBanner.tsx

```diff
@@ -4,7 +4,49 @@ import Link from 'docs/src/modules/components/Link';
 import ROUTES from 'docs/src/route';
 import FEATURE_TOGGLE from 'docs/src/featureToggle';
 
+const getSurveyMessage = () => (
+  <React.Fragment>
+    Influence the future of MUI!&nbsp;&nbsp;📫&nbsp;Please take a few minutes for the&nbsp;
```

Comment:
```suggestion
    🚀 Influence the future of MUI!&nbsp;&nbsp;&nbsp;Please take a few minutes for the&nbsp;
```

Very nit-picky ⎯ I'm sorry 😅 It's just because the mailbox emoji, from a distance, especially in a blue background, looks a bit muddy, it's hard to see it clearly. The rocket feels more upbeat?! 😬 

<img width="500" alt="Screen Shot 2022-12-12 at 09 31 54" src="https://user-images.githubusercontent.com/67129314/207046198-bd7534e1-3a1d-408d-9cee-8f7dc3e934a8.png">